openmdao/docs/examples/keplers_equation/keplers_equation.rst

```diff
@@ -26,22 +26,22 @@ to the balance, it solves the following equation:
 .. math::
      lhs(var) \cdot mult(var) = rhs(var)
 
-The _mult_ term is an optional multipler than can be applied to the
-left-hand-side (LHS) of the equation.  For our example we will assign the right-hand-side
-(RHS) to the mean anomaly (M) and the left-hand-side to :math:`E - e \sin{E}`
+The *mult* term is an optional multiplier than can be applied to the
+left-hand side (LHS) of the equation.  For our example, we will assign the right-hand side
+(RHS) to the mean anomaly (M), and the left-hand side to :math:`E - e \sin{E}`
```

Comment:
:math:`M` instead of M?

Reply:
What is your recommended practice for variables in text? Should we use :math:`M` always or `M` as in line 38?

Reply:
we should have a standard practice for this...do you like denoting variables with ":math:"  That could work.

Reply:
I agree we should have a standard practice but I'm not sure what that should be.. I suggested :math:`M` in this case because I saw the :math:`E - e \sin{E}`. What do you think?

Reply:
I just did the whole doc with :math: for all the vars, and it looks pretty good.  I'm not sure how many docs would be changed, but my next sweep can include that new look.  I like it.

Reply:
it's an especially consistent reference when you have an equation on display, like you do in this doc, to then later discuss the components of that equation using the same formatting.

Reply:
Sounds good!